app/src/main/java/ju/video/player/commons/FilterFiles.java

```diff
@@ -56,15 +59,33 @@ public List<String> getListFiles() throws IOException {
                 }
             } else {
                 BasicFileAttributes attributes = Files.readAttributes(file.toPath(), BasicFileAttributes.class);
-                System.out.println(file + " Media:" + verifyIsMediaFile(file) + " - Size:" + verifySize(attributes, initSize, endSize) + " - Date:" + verifyDate(attributes, initDate, endDate));
-                if (verifyIsMediaFile(file) && verifySize(attributes, initSize, endSize) && verifyDate(attributes, initDate, endDate)) {
+                if (verifyIsMediaFile(file) && verifySize(attributes, initSize, endSize)
+                        && verifyDate(attributes, initDate, endDate)
+                        && verifyFormatSelected(file, formatSelected)) {
                     listFilesName.add(fileName);
                 }
             }
         }
         return listFilesName;
     }
 
+    /**
+     * Verify the file belongs to the format selected in the Combobox.
+     *
+     * @param file
+     * @param formatSelected
+     * @return
```

Comment:
You could put the description of parameters and description of return

Reply:
Done